source/upgrade-guide/upgrading/restore_alerts.rst

```diff
@@ -79,9 +79,9 @@ Once the reindex of your alerts has finished, you can confirm that it was succes
 
 If everything worked well, the output will appear something like this:
 
-    .. code-block:: console
+    .. code-block:: none
+        :class: output
 
```

Comment:
It was correctly removed because it was duplicated: 

![curl](https://user-images.githubusercontent.com/60152567/76311109-84442700-62d0-11ea-9948-bbccd22ffff5.png)

Reply:
Noted